rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -628,11 +626,27 @@ private void saveSvpSpendTxHashUnsigned() {
             return;
         }
 
-        byte[] data = Optional.ofNullable(svpSpendTxHashUnsigned)
-            .map(BridgeSerializationUtils::serializeSha256Hash)
-            .orElse(null);
+        safeSaveToRepository(
+            SVP_SPEND_TX_HASH_UNSIGNED,
+            svpSpendTxHashUnsigned,
+            BridgeSerializationUtils::serializeSha256Hash);
+    }
+
+    private void saveSvpSpendTxWaitingForSignatures() {
+        if (!activations.isActive(RSKIP419)) {
+            return;
+        }
+
+        if (svpSpendTxWaitingForSignatures == null ||
```

Comment:
Not sure if we should check for null here, since we would need a default value when there is no svp ongoing right?

Reply:
Probably not, once the transaction is fully signed we want to clear this storage value by setting it to null. As we do with other values

Reply:
yes, careful here.
When there is no spend tx we should set the value to `new AbstractMap.SimpleEntry<>(null, null);`, and save it in the storage.
That's why we were not using the `safeSaveToRepository`, because we want the null values to be saved. And using the serializers throw a npe :)

Reply:
yes, marcos is right
When there is no spend tx we should set the value to new AbstractMap.SimpleEntry<>(null, null);, and save it in the storage.

In that case, you could do what was being done in the other methods:
```
        byte[] data = Optional.ofNullable(svpSpendTxWaitingForSignatures)
            .map(BridgeSerializationUtils::serializeRskTxWaitingForSignatures)
            .orElse(null);
```

Reply:
Makes sense I have updated to allow null value of the entry to be saved, as done in the other methods. 

> When there is no spend tx we should set the value to new AbstractMap.SimpleEntry<>(null, null), and save it in the storage.

This will not be allowed since it would throw a NPE. If the reference is not null it will try to serialize it, and since it contains null references inside it will throw a NPE. This is the reason why we cannot allow to save an "empty entry" 

Let me know if this looks good